spec/web/schema/squad_requests_spec.exs

```diff
@@ -0,0 +1,114 @@
+defmodule Squadster.Web.Schema.SquadRequestSpec do
+  use ESpec.Phoenix, async: true
+  use ESpec.Phoenix.Extend, :controller
+  use Phoenix.ConnTest
+
+  import Squadster.Support.Factory
+
+  alias Squadster.Formations.SquadRequest
+
+  let :create do
+    """
+      mutation createSquadRequest($squad_id: String) {
+        createSquadRequest(squad_id: $squad_id) {
+          insertedAt
+        }
+      }
+    """
+  end
+
+  let :delete do
+    """
+      mutation deleteSquadRequest($id: Int) {
+        deleteSquadRequest(id: $id) {
+          insertedAt
+        }
+      }
+    """
+  end
+
+  let :approve do
+    """
+      mutation approveSquadRequest($id: Int) {
+        approveSquadRequest(id: $id) {
+          approvedAt
+        }
+      }
+    """
+  end
+
+  let :create_squad_request_query, do: %{query: create(), variables: %{squad_id: squad().id}}
+
+  let :squad, do: insert(:squad)
+  let :user, do: insert(:user)
+
+  def delete_squad_request_query(squad_id) do
+    %{query: delete(), variables: %{id: squad_id}}
+  end
+
+  def approve_squad_request_query(squad_id) do
+    %{query: approve(), variables: %{id: squad_id}}
+  end
+
+  def api_request(payload) do
```

Comment:
Can we move this and the next methods to another module, something like `spec_helper` or `spec_helper_methods`?

Reply:
I think it have no sense because this function uses user from test context, if we move it to helper we'll need to pass it as a param, and it's not very convenient on my opinion. What do you think?